apps/api/src/domain/models/vocabulary-list/queries/vocabulary-list-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -90,108 +72,88 @@ const existingTerm: TermViewModel = TermViewModel.fromDto({
     contributions: [],
 });
 
-const vocabularyListEventStream = new TestEventStream()
-    .andThen<VocabularyListCreated>({
-        type: 'VOCABULARY_LIST_CREATED',
-        payload: {
-            name: vocabularyListName,
-            languageCodeForName: languageCodeForVocabularyListName,
-        },
-    })
-    .andThen<VocabularyListNameTranslated>({
-        type: 'VOCABULARY_LIST_NAME_TRANSLATED',
-    })
-    .andThen<VocabularyListFilterPropertyRegistered>({
-        type: 'VOCABULARY_LIST_PROPERTY_FILTER_REGISTERED',
-    })
-    .andThen<TermAddedToVocabularyList>({
-        type: 'TERM_ADDED_TO_VOCABULARY_LIST',
-        payload: {
-            termId,
-        },
-    })
-    .andThen<TermInVocabularyListAnalyzed>({
-        type: 'TERM_IN_VOCABULARY_LIST_ANALYZED',
-    })
-    .andThen<ResourcePublished>({
-        type: 'RESOURCE_PUBLISHED',
-    });
-
-// TODO check that contributors are joined in
+// TODO check that contributors are joined in- let's do this now
```

Comment:
Has this been done?

Reply:
![image](https://github.com/user-attachments/assets/e09c173a-0e22-46f8-af2f-ff323616ea1f)

Reply:
done